### PR TITLE
Weapon parser

### PR DESCRIPTION
**Pull request by howareyouman**
Added new class to parse weapons from text. Added some test of this parser. Weapon library will sooner grow up.

### CODE REVIEW

File: src/main/webapp/resources/weapons.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+gun
```
Comment (towelenee) [inline src/main/webapp/resources/weapons.txt]: Why this file twice in a repo?

Comment (howareyouman) [inline src/main/webapp/resources/weapons.txt]: this is for server and local tests

Comment (towelenee) [inline src/main/webapp/resources/weapons.txt]: Is it possible to have one file instead?

Comment (howareyouman) [inline src/main/webapp/resources/weapons.txt]: I'll check in a minute

Comment (howareyouman) [inline src/main/webapp/resources/weapons.txt]: Yes, it is!

File: src/main/java/com/w6/nlp/WeaponsParser.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+   LexicalizedParser parser;
+   
+   public WeaponsParser(LexicalizedParser globalParser) throws IOException
+   {
+       tokenizerFactory = PTBTokenizer.factory(new CoreLabelTokenFactory(),
+                        "invertible=true");
+       parser = globalParser;
+       try {
+           setWeaponsDictionary();
+       } catch (IOException e) {
```
Comment (towelenee) [inline src/main/java/com/w6/nlp/WeaponsParser.java:37]: Delete it, it does nothing

File: src/main/java/com/w6/nlp/WeaponsParser.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            InputStream in =
+                this.getClass().getResourceAsStream(globalPath);
+            BufferedReader br = new BufferedReader(new InputStreamReader(in));
+            String str;
+            while ((str = br.readLine()) != null) {
+                str = str.toLowerCase();
+                str.replaceAll("\\s+","");
+                weapons.add(str);
+            }
+        } catch (IOException e){
```
Comment (towelenee) [inline src/main/java/com/w6/nlp/WeaponsParser.java:57]: It does nothing as well

**Pull request merged**